src/sentry/static/sentry/app/views/settings/organizationDeveloperSettings/sentryApplicationDashboard/requestLog.tsx

```diff
@@ -0,0 +1,329 @@
+  errorsOnly: boolean;
+  currentPage: number;
+  requests: SentryAppWebhookRequest[];
+};
+
+export default class RequestLog extends AsyncComponent<Props, State> {
+  shouldReload = true;
+
+  get hasNextPage() {
+    return (this.state.currentPage + 1) * MAX_PER_PAGE < this.state.requests.length;
+  }
+
+  get hasPrevPage() {
+    return this.state.currentPage > 0;
+  }
+
+  getEndpoints(): Array<[string, string, any] | [string, string]> {
+    const {slug} = this.props.app;
+
+    const query: any = {};
+    if (this.state) {
+      if (this.state.eventType !== ALL_EVENTS) {
+        query.eventType = this.state.eventType;
+      }
+      if (this.state.errorsOnly) {
+        query.errorsOnly = true;
+      }
+    }
+
+    return [['requests', `/sentry-apps/${slug}/requests/`, {query}]];
+  }
+
+  getDefaultState() {
+    return {
+      ...super.getDefaultState(),
+      requests: [],
+      eventType: ALL_EVENTS,
+      errorsOnly: false,
+      currentPage: 0,
+    };
+  }
+
+  handleChangeEventType = (eventType: string) => {
+    this.setState({
+      eventType,
+      currentPage: 0,
+    });
+    this.remountComponent();
+  };
+
+  handleChangeErrorsOnly = () => {
+    this.setState({
+      errorsOnly: !this.state.errorsOnly,
+      currentPage: 0,
+    });
+    this.remountComponent();
+  };
+
+  handleNextPage = () => {
+    this.setState({
+      currentPage: this.state.currentPage + 1,
+    });
+  };
+
+  handlePrevPage = () => {
+    this.setState({
+      currentPage: this.state.currentPage - 1,
+    });
+  };
+
+  renderLoading() {
+    return this.renderBody();
+  }
+
+  renderBody() {
+    const {requests, eventType, errorsOnly, currentPage} = this.state;
+    const {app} = this.props;
+
+    const currentRequests = requests.slice(
+      currentPage * MAX_PER_PAGE,
+      (currentPage + 1) * MAX_PER_PAGE
+    );
+
+    return (
+      <React.Fragment>
+        <h4>{t('Request Log')}</h4>
+
+        <div>
+          <p>
+            {t(
+              'This log shows the status of any outgoing webhook requests from Sentry to your integration.'
+            )}
+          </p>
+
+          <RequestLogFilters>
+            <DropdownControl
+              label={eventType}
+              menuWidth="220px"
+              buttonProps={{style: {zIndex: theme.zIndex.header - 1}}}
+            >
+              {getEventTypes(app).map(type => (
+                <DropdownItem
+                  key={type}
+                  onSelect={this.handleChangeEventType}
+                  eventKey={type}
+                  isActive={eventType === type}
+                >
+                  {type}
+                </DropdownItem>
+              ))}
+            </DropdownControl>
+
+            <Button
+              onClick={this.handleChangeErrorsOnly}
+              priority={errorsOnly ? 'primary' : 'default'}
+            >
+              <ErrorsOnlyCheckbox>
+                <Checkbox onChange={() => {}} checked={errorsOnly} />
```

Comment:
@billyvg trie to remove it, and it doesn't work because React expects it for all `input`s so it's not even a prop type we can really remove. I think the best option for now is just to make  the onChange a no-op, in the future I might even remove this checkbox filter component thing completely anyway